tastypie/authentication.py

```diff
@@ -37,6 +39,9 @@ class Authentication(object):
 
     By default, this indicates the user is always authenticated.
     """
+    def __init__(self, require_active=True):
```

Comment:
spaces inside the parens here

Reply:
old and awesome habits die hard

Reply:
@cullenbmacdonald readability counts man.